apps/central-scan/backend/src/fujitsu_scanner.test.ts

```diff
@@ -115,9 +115,9 @@ test('fujitsu scanner can scans with expected params on letter size election', (
     'scanimage',
     expect.arrayContaining([
       '--page-width',
-      '215.872',
+      '215.9',
       '--page-height',
-      '336.506',
+      '336.55',
```

Comment:
We expectedly don't see a meaningful change in the height param for letter HMPBs since, as noted in the PR description, we specify the BMDB height for that case (13.25 inches > 11 inches)